crates/ruff_python_parser/src/lexer.rs

```diff
@@ -538,19 +547,19 @@ impl<'source> Lexer<'source> {
         #[cfg(debug_assertions)]
         debug_assert_eq!(self.cursor.previous(), quote);
 
-        let mut flags = FStringContextFlags::empty();
+        let mut flags = StringFlags::default().with_f_prefix().unwrap();
         if quote == '"' {
-            flags |= FStringContextFlags::DOUBLE;
+            flags = flags.with_double_quotes();
         }
         if is_raw_string {
-            flags |= FStringContextFlags::RAW;
+            flags = flags.with_r_prefix().unwrap();
```

Comment:
Should this error be propagated? Or is it handled elsewhere / upstream?

Reply:
This _shouldn't_ ever error -- the only reason this would error out is if it would be invalid to combine the `f` and `r` prefixes. But _we_ know that it's always valid to combine the `f` and `r` prefixes. So calling `unwrap()` here should be safe.

But I should probably change it to an `expect()` call, so that it's more obvious why it's safe.